snuba/clickhouse/native.py

```diff
@@ -25,13 +25,17 @@ def __init__(
         self,
         host: str,
         port: int,
+        user: str,
+        password: str,
```

Comment:
@chhetripradeep It seems in the latest change you are no longer passing the user and password into here in cleanup and optimize. But these are required arguments now so won't that just fail? In those cases I think what we want to do is actually fetch the user and password out of the cluster and still pass it into here. Do you agree?

Reply:
@lynnagara ahh you are right, my bad. Let me fix it. Thank you.